.github/renovate.json5

```diff
@@ -76,9 +76,9 @@
     "matchPackageNames": ["/.*/"],
     "groupName": "All updates",
     "automerge": true,
-    // A known issue is that some non-Konflux CI jobs in currently fail, which may prevent successful auto-merging with a "branch" auto-merge setting.
-    // Therefore, we use PR merge type and have automation approve PRs.
-    "automergeType": "pr",
+    // PRs can't be actually automerged because we require approval from CODEOWNERS which Renovate can't bypass,
+    // therefore, we set automerge type to branch.
+    "automergeType": "branch",
     "automergeStrategy": "squash",
     // Tell Renovate that it can automerge branches at any time of the day.
     "automergeSchedule": [
```

Comment:
[not about this line] When you use `"automergeType": "branch"`, you don't need `"labels": ["auto-approve"]` because `labels` is what should be put on the PR.

See https://docs.renovatebot.com/configuration-options/#automergetype
Technically Renovate/MintMaker can open a PR eventually even if you have `branch` type, but you'd want to look at that PR.

Similarly, you don't need `"platformAutomerge": true`.